ui/cypress/fixtures/config.json

```diff
@@ -1,12 +1,13 @@
 {
-    "url": "/api/kubernetes",
-    "url_salt": "/api/salt",
-    "url_prometheus": "/api/prometheus",
-    "url_grafana": "/grafana",
-    "url_doc": "/docs",
-    "url_alertmanager": "/api/alertmanager",
-    "url_navbar": "/shell/solution-ui-navbar.1.0.0.js",
-    "url_alerts": "/shell/alerts.1.0.0.js",
-    "alerts_lib_version": "1.0.0",
-    "url_navbar_config": "/shell/config.json"
+  "url": "/api/kubernetes",
+  "url_salt": "/api/salt",
+  "url_prometheus": "/api/prometheus",
+  "url_grafana": "/grafana",
+  "url_doc": "/docs",
+  "url_alertmanager": "/api/alertmanager",
+  "url_navbar": "/shell/solution-ui-navbar.1.0.0.js",
+  "url_alerts": "/shell/alerts.1.0.0.js",
+  "alerts_lib_version": "1.0.0",
+  "url_navbar_config": "/shell/config.json",
+  "url_support": "https://support.scality.com/hc/en-us"
```

Comment:
don't see why we need a real link here. 
And please add an empty entry in metalk8s UI config.json 
`"url_support": ""`

Reply:
Actually, if we could have something like `https://github.com/scality/metalk8s/discussions/new`, that'd be awesome!